bridge_adaptivity/module/templates/module/widgets/select_policy.html

```diff
@@ -0,0 +1,15 @@
+{% load bootstrap3 %}
+<div style="width:100%;">
+  <select style="width:50%; display:inline-block; " class="policy_select form-control" name="{{ widget.name }}"{% include "django/forms/widgets/attrs.html" %}>
```

Comment:
I would use 120 column limit for the code style of the .html files also to make the review process more comfortable.